Doc/library/typing.rst

```diff
@@ -1232,8 +1232,12 @@ These are not used in annotations. They are building blocks for creating generic
 
    .. note::
 
-        :func:`runtime_checkable` will check only the presence of the required methods,
-        not their type signatures.
+        :func:`runtime_checkable` will check only the presence of the required
+        methods, not their type signatures. For example, :class:`ssl.SSLObject`
+        implements :func:`__init__`, therefore it passes an :func:`issubclass`
```

Comment:
I think the directive for `__init__` is `meth`, but I don't recall if using `func` makes a difference. So feel free to ignore this.
```suggestion
        implements :meth:`__init__`, therefore it passes an :func:`issubclass`
```

Reply:
I saw that too :) I kept it from the original example because it worked in my testing, but you are right, it's better to clean it up.